cerebrium/test-and-deploy.mdx

```diff
@@ -106,20 +123,33 @@ You can see an example of the request and response objects for calls made to you
 <ParamField header="Authorization" type="string" required>
   This is the Cerebrium API key used to authenticate your request. You can get it from your Cerebrium dashboard.
 </ParamField>
+<ParamField body="Content-Type" type="string" required>
+   The content type of your request. Must be application/json.
+</ParamField>
 <ParamField body="data" type="array" required>
-   The data you would like to send to your model. You may feed an ndarray or Tensor directly but it must be within an array.
+   A list of data points you would like to send to your model. e.g. for 1 data point of 3 features: [[1,2,3]].
 </ParamField>
 
 <ResponseExample>
 
 ```json Response
-[<DATA>]
+{
+  "result": [<MODEL_PREDICTION>],
+  "run_id": "<RUN_ID>",
+  "prediction_ids": ["<PREDICTION_ID>"]
```

Comment:
Is this the response object even for custom models?

Reply:
Correcto, it has been changed